_data/toc.yaml

```diff
@@ -157,6 +155,10 @@ guides:
       title: Manage data in containers
   - path: /engine/docker-overview/
     title: Docker overview
+  - path: /release-notes/docker-ce/
```

Comment:
Maybe during an upgrade, but not when they're installing for the first time. 

Can you tell me about a time when you needed to read the release notes before understanding what a product does and what problems it solves?

I always go back to this analogy. You have a new fancy car that you believe it's better than any other car out there. Are you going to take me for a test drive and **show me**, or are you going to tell me the story of how you just introduced seats in the latest release, and a steering wheel two versions ago? Do I even care about that story if I'm just learning about your car for the first time?